Description: Patch for the newplugin-script, to not require a normal
 vdr-source-dir
Author: Thomas Schmidt <tschmidt@debian.org>
Re-Authored: Lars Hanisch <dvb@flensrocker.de>

--- a/newplugin
+++ b/newplugin
@@ -24,7 +24,7 @@
 $PLUGIN_DESCRIPTION = "Enter description for '$PLUGIN_NAME' plugin";
 $PLUGIN_MAINENTRY = $PLUGIN_CLASS;
 
-$PLUGINS_SRC = "PLUGINS/src";
+$PLUGINS_SRC = "./";
 
 $README = qq
 {This is a "plugin" for the Video Disk Recorder (VDR).
@@ -172,7 +172,7 @@
 	\@-rm -rf \$(TMPDIR)/\$(ARCHIVE)
 	\@mkdir \$(TMPDIR)/\$(ARCHIVE)
 	\@cp -a * \$(TMPDIR)/\$(ARCHIVE)
-	\@tar czf \$(PACKAGE).tgz -C \$(TMPDIR) \$(ARCHIVE)
+	\@tar czf \$(PACKAGE).tgz -C \$(TMPDIR) --exclude debian --exclude CVS --exclude .svn --exclude .hg --exclude .git \$(ARCHIVE)
 	\@-rm -rf \$(TMPDIR)/\$(ARCHIVE)
 	\@echo Distribution package created as \$(PACKAGE).tgz
 
@@ -346,7 +346,7 @@
 * fill in the code skeleton in "$PLUGIN_NAME.c" to implement your plugin function
 * add further source files if necessary
 * adapt the "Makefile" if necessary
-* do "make plugins" from the VDR source directory to build your plugin
+* do "make all" from the plugin's source directory to build your plugin
 
 };
 
